res_partner_attributes_add_BN/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Business identification Number in Canada (BN/NE)
+================================================
+
+Add the Business identification Number (BN/NE) to the partner form.
+
+Installation
+============
+
+
+Configuration
+=============
+
+
+Usage
+=====
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+
```

Comment:
"Go to a partner form and fill in the Business identification Number, located under the email"